services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -248,9 +262,14 @@ public void initMetrics() {
         this::getFinalizedEpochValue);
     metricsSystem.createGauge(
         ArtemisMetricCategory.BEACON,
-        "justified_epoch",
+        "current_justified_epoch",
         "Current justified epoch",
         this::getJustifiedEpochValue);
+    metricsSystem.createGauge(
+        ArtemisMetricCategory.BEACON,
+        "previous_justified_epoch",
+        "Current previously justified epoch",
```

Comment:
Would "Previously justified epoch" make more sense?

Reply:
i just copied the description from the spec...